Add CPU and mem calculation to mesh-dashboard

mandarjog opened the pull request:
* Istio cpu cost per 1k requests
* Istio memory costs - This is a bit tricky to include 1k requests across the board. Mixer is normalized with number of requests. Proxies are normalized across number of proxies.

https://snapshot.raintank.io/dashboard/snapshot/qftZxEuZZdZKdHGpemm0gJO9EUN5633O

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **mandarjog**

If they are not already assigned, you can assign the PR to them by writing `/assign @mandarjog` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[install/OWNERS](https://github.com/istio/istio/blob/release-1.0/install/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["mandarjog"]} -->

mandarjog commented:
@josenativio @suryadu

douglas-reid commented on install/kubernetes/helm/istio/charts/grafana/dashboards/istio-mesh-dashboard.json:
```diff
@@ -363,6 +411,216 @@
+          "expr": "sum(rate(container_cpu_usage_seconds_total{pod_name=~\"istio-telemetry-.*\"}[1m]))/ (round(sum(irate(istio_requests_total[1m])), 0.001)/1000)",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "telemetry",
```
I think we want to make these names match their deployment/workload/container names: istio-telemetry, istio-ingressgateway, istio-proxy, istio-policy.

douglas-reid commented on install/kubernetes/helm/istio/charts/grafana/dashboards/istio-mesh-dashboard.json:
```diff
@@ -363,6 +411,216 @@
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Istio CPU per 1k requests",
```
I think we can drop "Istio" from the title here. Just "CPU per 1k requests".

mandarjog commented:
1. Moved the charts at the bottom of the page.
2. Changed metrics names to match deployments.
3. Dropped istio from Panel name.
4. I would defer on creating a new dashboard.

mandarjog commented:
@douglas-reid updated dashboard is here: https://snapshot.raintank.io/dashboard/snapshot/kKaST16YaxyvIXhU2iJqFLWjGt8MOKqy

mandarjog commented:
closing this in favor of #9129
